omnibus/files/private-chef-cookbooks/private-chef/libraries/nginx_erb.rb

```diff
@@ -106,4 +106,14 @@ def access_log(proto)
       '/var/log/opscode/nginx/access.log'
     end
   end
+
+  def time_format
+    time_format = node['private_chef']['nginx']['time_format']
+    if time_format != 'time_iso8601' and time_format != 'time_local'
```

Comment:
Minor suggestion: unless can be used in place of if !=

`unless( time_format == 'time_iso8601' or time_format == 'time_local')`

Reply:
sure, will make the changes. Thanks for the suggestion!